.github/workflows/ros2_build_test/entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh -l
+. /opt/ros/humble/setup.sh
+
+colcon build --continue-on-error --packages-skip alica_ros_proxy alica_tracing alica_ros_turtlesim alica_tests supplementary_tests alica_msgs 2> result.txt
```

Comment:
If you `tee` to text file then you can get real time logs from CI while also being able to cat them later.  Can also check what the return value of this is, since we might be able to just check `$?` and exit based on it, instead of grepping from a file.

Reply:
> Why --continue-on--error?

This just means that the build won't terminate immediately if one package fails, so if multiple things are failing we can see them all at once

Reply:
> If you `tee` to text file then you can get real time logs from CI while also being able to cat them later. Can also check what the return value of this is, since we might be able to just check `$?` and exit based on it, instead of grepping from a file.

https://colcon.readthedocs.io/en/released/reference/verb/build.html - The docs don't have anything as to what variable would store that value - there's documentation on how to get colcon test to spit out a return value, but nothing for build that I saw